packages/allure-cypress/src/index.ts

```diff
@@ -0,0 +1,188 @@
+    }
+  }
+}
+
+export type CypressWrappedAttachment = { type: string; data: unknown };
+
+export const label = (name: string, value: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      labels: [{ name, value }],
+    },
+  });
+};
+export const link = (url: string, name?: string, type?: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      links: [{ type, url, name }],
+    },
+  });
+};
+export const parameter = (name: string, value: string, options?: ParameterOptions) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      parameter: [{ name, value, ...options }],
+    },
+  });
+};
+export const description = (markdown: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      description: markdown,
+    },
+  });
+};
+export const descriptionHtml = (html: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      descriptionHtml: html,
+    },
+  });
+};
+export const testCaseId = (value: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      testCaseId: value,
+    },
+  });
+};
+export const historyId = (value: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      historyId: value,
+    },
+  });
+};
+export const allureId = (value: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      labels: [{ name: LabelName.ALLURE_ID, value }],
+    },
+  });
+};
+export const displayName = (name: string) => {
+  cy.now("allureReporterMessage", {
+    type: MessageType.METADATA,
+    payload: {
+      displayName: name,
+    },
+  });
+};
+export const issue = (url: string, name?: string) => {
+  link(url, name, LinkType.ISSUE);
+};
+export const tms = (url: string, name?: string) => {
+  link(url, name, LinkType.TMS);
+};
+export const epic = (name: string) => {
+  label(LabelName.EPIC, name);
+};
+export const feature = (name: string) => {
+  label(LabelName.FEATURE, name);
+};
+export const story = (name: string) => {
+  label(LabelName.STORY, name);
+};
+export const suite = (name: string) => {
+  label(LabelName.SUITE, name);
+};
+export const parentSuite = (name: string) => {
+  label(LabelName.PARENT_SUITE, name);
+};
+export const subSuite = (name: string) => {
+  label(LabelName.SUB_SUITE, name);
+};
+export const owner = (name: string) => {
+  label(LabelName.OWNER, name);
+};
+export const severity = (name: string) => {
+  label(LabelName.SEVERITY, name);
+};
+export const layer = (name: string) => {
+  label(LabelName.LAYER, name);
+};
+export const tag = (name: string) => {
+  label(LabelName.TAG, name);
+};
+export const attachment = (name: string, content: unknown, type: string, encoding: string = "utf8") => {
```

Comment:
Not sure about stability because `BufferEncoding` is a node type and it won't be available when typescript environment doesn't have types for node. But let's add and see